ext/ldap/ldap.c

```diff
@@ -53,9 +53,7 @@
 #include "ext/standard/php_string.h"
 #include "ext/standard/info.h"
 
-#ifdef HAVE_LDAP_SASL_H
-#include <sasl.h>
-#elif defined(HAVE_LDAP_SASL_SASL_H)
+#ifdef HAVE_LDAP_SASL
```

Comment:
Do you know why this used to check for `sasl.h` as well? Did this change in some specific version?

Reply:
@nikic, A long time ago, `sasl.h` was located in `includedir`. That header (and others) now live in `includedir/sasl`. I don't know exactly when this changed, but it was at least 12 years ago (probably closer to 15). PHP 5.0 (July 2009) looks to be the first use of the conditional logic.

I've never actually seen `sasl.h` in `includedir`, only in `includedir/sasl`.

Reply:
Thanks for checking, in that case this looks all good.